### ISSUE TITLE
camelcase: ignoreImports should also ignore usage of imported identifiers

### ISSUE THREAD

**Issue by privatenumber** [privatenumber/lintroll#93]
The `camelcase` rule with `ignoreImports: true` ignores the import statement but still flags usage of the imported type:

```ts
import { type Unstable_DevWorker } from 'wrangler';  // OK - import ignored

export type WorkerFixture = {
  worker: Unstable_DevWorker;  // ERROR - type usage flagged
};
```

The `ignoreImports` option should also ignore usages of imported identifiers, not just the import statement itself. Otherwise users still need disable comments for every usage.

**Expected behavior**: When an identifier is imported and `ignoreImports: true` is set, all references to that identifier should be ignored.

**Actual behavior**: Only the import statement is ignored; usages of the imported identifier are still flagged.
